StoreTestWPF.ViewModel/ViewModels/CakeViewModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿using StoreTestWPF.DAL.Models;
+
+namespace StoreTestWPF.ViewModel.ViewModels
+{
+    public class CakeViewModel : ViewModelBase
+    {
+        private Cake cake;
+
+        public CakeViewModel(Cake cake)
+        {
+            this.cake = cake;
+        }
+
+        public Cake Cake { get { return this.cake; }}
+
+        public string Title
+        {
+            get { return this.cake.Title; }
+            set
+            {
+                this.cake.Title = value;
```

Comment:
When we set a new value in ViewModel we should make sure that the value differs from existing value in field. Notifying is pretty costly, so we should avoid unnecessary calls.

Reply:
Please change in all places.

Reply:
Fixed